node/cdn/src/blocks.rs

```diff
@@ -138,7 +138,7 @@ impl CdnBlockSync {
                 debug!("Synced the ledger up to block {completed_height}");
 
                 // Retrieve the latest height, according to the ledger.
-                let node_height = cow_to_copied!(ledger.vm().block_store().heights().max().unwrap_or_default());
+                let node_height = ledger.vm().block_store().heights().max().unwrap_or_default().into_owned();
```

Comment:
```suggestion
                let node_height = *ledger.vm().block_store().heights().max().unwrap_or_default();
```
should be fine as long it's just the usual `u32`